File: src/dataset/c/two_fer_Exercism_ThomasZumsteg/gemini/GeminiFlash_two_fer_v2.c
#include "two_fer.h"
#include <stdio.h>
#include <string.h>

void two_fer(char *buffer, const char *name) {
    const char *name_ptr = (name != NULL) ? name : "you";
    snprintf(buffer, 256, "One for %s, one for me.", name_ptr);
}

File: src/dataset/Java/two_fer_Schenz/gemini/GeminiFlash_Twofer_v2.java
class Twofer {
    private static final String TWOFER_DEFAULT = "One for you, one for me.";

    String twofer(String name) {
        if (name == null || name.trim().isEmpty()) {
            return TWOFER_DEFAULT;
        }

        return "One for " + name + ", one for me.";
    }
}